website/docs/d/data_lake_pipeline_run.html.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: data_lake_pipeline_run"
+sidebar_current: "docs-mongodbatlas-datasource-data-lake-pipeline-run"
+description: |-
+    Describes a Data Lake Pipeline Run.
+---
+
+# Data Source: mongodbatlas_data_lake_pipeline_run
+
+`mongodbatlas_data_lake_pipeline_run` describe a Data Lake Pipeline Run.
+
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```terraform
+resource "mongodbatlas_data_lake_pipeline" "pipeline" {
+      //assuming we've already setup project and cluster in another block
+      project_id       = mongodbatlas_project.projectTest.project_id
+      name             = "DataLakePipelineName"
+      sink {
+        type = "DLS"
+        partition_fields {
+            name = "access"
+            order = 0
+        }
+      }
+      source {
+        type = "ON_DEMAND_CPS"
+        cluster_name = mongodbatlas_cluster.clusterTest.name
+        database_name = "sample_airbnb"
+        collection_name = "listingsAndReviews"
+      }
+      transformations {
+              field = "test"
+              type  = "EXCLUDE"
+      }
+      transformations {
+              field = "test22"
+              type  = "EXCLUDE"
+      }
+}
+
+data "mongodbatlas_data_lake_pipeline_runs" "pipeline_run" {
```

Comment:
Any benefit of including both data sources in this example? if not suggest would only show the singular version here.

Reply:
It was discussed in this thread: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1177#discussion_r1202739924